dataset-documentation/DATASETDOC-fall25-teamA.md

```
@@ -0,0 +1,114 @@
+* Who are the client contacts for the project?
+
+David Gasser: dgasser@chapa.org <br>
+Josh Vogel: jvogel@chapa.org
+* What class was this project part of?
+  
+DS 701 at Boston University <br>
+
+***Dataset Information***
+
+* What data sets did you use in your project? Please provide a link to the data sets, this could be a link to a folder in your GitHub Repo, Spark\! owned Google Drive Folder for this project, or a path on the SCC, etc.
+[Link](https://drive.google.com/drive/u/1/folders/1nlVp39HIudp6QNPv4IvXQCVbhfo7djES),
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)